insights/client/connection.py

```diff
@@ -216,9 +220,10 @@ def user_agent(self):
                 logger.warning("Failed to detect OS version: %s", e)
         kernel_version = "%s %s" % (platform.system(), platform.release())
 
-        ua = "{client_version} ({core_version}; {requests_version}) {os_family} {os_release} ({python_version}; {kernel_version})".format(
+        ua = "{client_version} ({core_version}; {requests_version}) {os_family} {os_release} ({python_version}; {kernel_version}); {invocation}".format(
```

Comment:
Can we name this thing more technically accurate and leave the assumption about "invocation source" to the interpretation? Something like "parent_process"? Since we aren't claiming that the parent process definitively specifies where the invocation source is coming from.

Reply:
this is a good idea. I'll update it